frontend/Dockerfile

```diff
@@ -1,10 +1,15 @@
 FROM node:13.2-alpine
 
+ENV NUXT_PORT 3000
+
+CMD ["/usr/local/bin/npm", "start"]
```

Comment:
In `docker-compose.prod.yml`, it is necessary to run `npm start`.

Reply:
Not sure what you're referring to :) For prod we're just going with the `CMD` that is in this `Dockerfile`, for the dev compose file we're launching `/src/frontend/dev-nuxt.sh`.